src/main/java/org/broadinstitute/hellbender/engine/ReadWalker.java

```diff
@@ -65,8 +67,12 @@ public void traverse() {
                           new ReferenceContext(reference, readInterval), // Will create an empty ReferenceContext if reference or readInterval == null
                           new FeatureContext(features, readInterval));   // Will create an empty FeatureContext if features or readInterval == null
                 });
+
+        // TODO: where should the summary output go ?
+        String filterSummary = countedFilter.getFilteredCount() != 0 ? countedFilter.getSummaryLine() : "No reads filtered by " + countedFilter.getName();
```

Comment:
Could you also make `countedFilter.getSummaryLine()` return the `"No reads filtered by " + getName()` String automatically if `getFilteredCount()` == 0?